common/sepolicy/vendor/file_contexts

```diff
@@ -27,3 +27,5 @@
 /dev/block/by-name/metadata                         u:object_r:metadata_block_device:s0
 /dev/block/by-name/super                            u:object_r:super_block_device:s0
 /dev/block/by-name/misc                             u:object_r:misc_block_device:s0
+
+/dev/loop16                                         u:object_r:system_block_device:s0
```

Comment:
Hi @MaksymPrymierov ,

This element is already initialized with loop_device here:
https://cs.android.com/android/platform/superproject/+/master:system/sepolicy/private/file_contexts;l=88;drc=6afdb72cbb96dee8b8372274a49416a7fa0594ce

Which process does need this rule? Can you attach corresponding dmesg fail messages to every commit message/rule you are adding?

Reply:
Hi @rsglobal, Yes, I can add selinux warnings to commit messages.

I cheched this on an OrangePI PC device and the device `/dev/loop16` is in the blk_file context.

Related  warnings for this commit:
```
audit: type=1400 audit(10.900:10): avc:  denied  { create } for  pid=28 comm="kdevtmpfs" name="loop16" scontext=u:r:kernel:s0 tcontext=u:object_r:device:s0 tclass=blk_file permissive=1

audit: type=1400 audit(10.900:11): avc:  denied  { setattr } for  pid=28 comm="kdevtmpfs" name="loop16" dev="devtmpfs" ino=13317 scontext=u:r:kernel:s0 tcontext=u:object_r:device:s0 tclass=blk_file permissive=1
```